packages/phishing-controller/CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0]
+### Changed
+- **BREAKING:** Refactor to Cost-Optimized Phishing List Data Architecture. ([#1080](https://github.com/MetaMask/core/pull/1080))
+  - Rather than periodically downloading two separate configurations (MetaMask and Phishfort), we now download a combined "stalelist" and "hotlist". The stalelist is downloaded every 4 days, and the hotlist is downloaded every 30 minutes. The hotlist only includes data from the last 8 days, which should dramatically reduce the required network traffic for phishing config updates.
+  - When a site is blocked, we no longer know which list is responsible due to the combined format. We will need to come up with another way to attribute blocks to a specific list; this controller will no longer be responsible for that.
+  - This change includes the removal of the exports:
+    - `METAMASK_CONFIG_FILE` and `PHISHFORT_HOTLIST_FILE` (replaced by `METAMASK_STALELIST_FILE` and `METAMASK_HOTLIST_DIFF_FILE`)
+    - `METAMASK_CONFIG_URL` and `PHISHFORT_HOTLIST_URL` (replaced by `METAMASK_STALELIST_URL` and `METAMASK_HOTLIST_DIFF_URL`)
+    - `EthPhishingResponse` (replaced by `EthPhishingDetectConfig` for the API response and `PhishingStalelist` for the list in controller state, as they're now different)
```

Comment:
Should this be:

``` suggestion
- `EthPhishingResponse` (replaced by `PhishingStalelist` for the list in controller state, as it's now different)
```

Reply:
This entry is for a removed export, so that indentation level makes sense. It was split into two types as well, one for the API response and one for the state.

I see I got the type names wrong though:

```suggestion
    - `EthPhishingResponse` (replaced by `PhishingStalelist` for the API response and `PhishingListState` for the list in controller state, as they're now different)
```

Reply:
Let me know if that makes sense

Reply:
Oops, sorry, I didn't realize I'd removed the indentation (sometimes GitHub doesn't capture the indentation when I copy; not sure why). That change makes sense.